### DISCUSSION TITLE
How to work on features

### DISCUSSION THREAD

**Discussion by MMikhailova** [lab-brussels-1/The-first-group#36] [Announcements]
**One team member** will create **one feature**.  
For each feature he/she will create **two issues** and one PR.  

Example:

Minju is doing Title. She creates 2 issues, one for html and one for css.
Then she creates one branch called Minju_Title and make  one PR to merge this feature to main. This PR should be linked to both issues (html&css)

@minjupgeorge @zerubabel4 @Carsiti

**Comment by colevandersWands** [comment]
Close!  It's simplest to have just one issue per feature with `html` and `css` labels. After merging the initial PR you can always create new ones to add to the feature.

Where was it not clear in the instructions?  I can update this

**Reply by MMikhailova** [reply]
Hi @colevandersWands ! 

May be I got it wrong but I wrote it based of what Samir explained here [in discord](https://discord.com/channels/894607053367181402/946371706732294175/955557299483738183).
and Tamer here [in discord](https://discord.com/channels/894607053367181402/944636735625764894/955254956137525288)

As I understood  from Tamir's message "Issues, branches and PRs are free 🤩  so you can create as much as you want." 
So now I am very confused...

**Reply by colevandersWands** [reply]
I think I answered this in your [check-in](https://github.com/lab-brussels-1/home/issues/220), let me know if it still isn't clear
